src/components/Wizard/Wizard.tsx

```diff
@@ -1,19 +1,15 @@
-import React, { FC, ReactNode } from 'react';
+import React, { FC } from 'react';
 
-// Style Imports
 import classNames from 'classnames';
 
-// Component Imports
+import { Buttons, Confirmation, Loading } from './Presets';
+
 import { InfoTooltip } from '../InfoTooltip/InfoTooltip';
 
-// Preset Screens
-import { Buttons } from './Presets/Buttons';
-import { Confirmation } from './Presets/Confirmation';
-import { Loading } from './Presets/Loading';
 import './Wizard.scss';
 
 export interface HeaderProps {
-  header?: ReactNode;
+  header?: React.ReactNode;
```

Comment:
Only added this as we import React due to lint - 'React' must be in scope when using JSX